apps/client/ui/Range/Range.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect, useRef } from 'react'
+import type { FC } from 'react'
+import 'rc-slider/assets/index.css';
+import Slider from 'rc-slider';
+
+type CustomRangeProps = {
+  min?: number
+  max?: number
+  step?: number
+  defaultValue?: number[]
+  className?: string
+};
+
+const sliderStyles = ''
+
+export const CustomRange: FC<CustomRangeProps> = ({
+  min = 0,
+  max = 10,
+  step = 1,
+  defaultValue = [0, 10],
+  className,
+  ...props
+}) => {
+  const [values, setValues] = useState<number[]>(defaultValue);
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  const handleOnChange = (newValues: number[]) => {
+    setValues(newValues);
+  };
+
+  const handleTooltipPosition = () => {
```

Comment:
I'm sure you don't need this calculations, `containerRef` and `useEffect` below 🕵️ you don't need to pass absolute values like `px` in this case - quite better would be to calculate percentage value and pass it declaratively to the correct element 👌 it's quite simpler and more React way (because it's declarative and not imperative like in your solution - React doesn't like it if not necessary). Let me know if I'm not clear enough 🙌 

Another thing - you can use `handleRender` prop in `Slider` to display tooltip (like on the pic) 👌 but your solution should work as well 🤝 you can see also calculation of percentage position (line 106) and small style difference than in your example (line 104 - `absolute -translate-x-1/2` classes)
<img width="972" alt="Zrzut ekranu 2023-09-24 o 21 19 49" src="https://github.com/darionara/Movies-App/assets/30324689/d3687b21-6dea-4e89-b169-baa44afe423d">